EIPS/eip-5190.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 5190
+title: Account-bound Finance
+description: A smart contract design pattern in decentralized finance to remove insolvency and arbitrary loss of funds using blockchain technology only. Extension of EIP-4973
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155, 4973
```

Comment:
EIP-4973 is in Draft. This cannot be moved to a status beyond that of EIP-4973.

Reply:
If EIP-4973 doesn't move out of draft, this EIP won't be able to move out of draft as well.  If you are OK with this constraint, that is fine, but be aware that references in this EIP should only be to EIPs that are required reading for this EIP.